proposals/Level-1.md

```diff
@@ -78,94 +77,69 @@ instruction. Thrown exceptions are handled as follows:
 1. Throws not caught within a function body continue up the call stack, popping
    call frames, until an enclosing try block is found.
    
-1. If the call stack is exhausted without any enclosing try blocks, embedder
+1. If the call stack is exhausted without any enclosing try blocks, the embedder
    defines how to handle the uncaught exception.
    
 ### Event handling
```

Comment:
We talked about this offline some, but I'll mention it here in case other people have suggestions: I'm not sure "event" is the right term here, since it evokes the idea of events in the `addEventHandler` sense. I suggested something like "condition," "effect," or maybe "signal" but I'm not convinced these are better terms.

Reply:
Btw, I'm a bit confused about the diff here. It seems like an earlier version of that section has already landed?